source/docs/articles/sites/varnish/caching-advancedtopics.md

```diff
@@ -142,7 +142,7 @@ If you're using the Environment Access: Locked security setting on a site envir
 
 ## Pantheon's .vcl File
```

Comment:
@ari-gold I thought about that. Any links to the existing heading would need to be changed, unless we can redirect to a different heading in the same document. The last time I tried that it failed in Drupal.